src/view/InputView.js

```diff
@@ -0,0 +1,25 @@
+import Console from "../util/Console.js";
+
+const InputView = {
+  async inputMoney(query) {
+    const buyMoney = await Console.read(query);
+    return buyMoney;
```

Comment:
```suggestion
return Console.read(query);
```

위와 같이 간소화한다면, 아래에 정의한 메서드도 똑같이 간소화할 수 있지 않나요?
그렇다면, 굳이 InputView의 메서드를 여러 개 작성할 필요가 있을까요?

Reply:
Console.read(query)의 리턴값이 나올 때까지 await를 이용해서 기다렸다 값이 나오면 그 값을 리턴하고자 한 코드였는데, 
저렇게 바로 리턴을 해도 되는군요!! 알맞게 수정해서 다시 반영해보겠습니다!

Reply:
아직 수정이 안 된 부분일까요?